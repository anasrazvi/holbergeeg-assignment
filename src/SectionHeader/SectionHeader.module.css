.sectionHeader{
    font-weight: 500;
    text-align: left;
    padding: 15px 0 15px 10px;
    font-size: 18px;
    background-color: #bbbb;
  }