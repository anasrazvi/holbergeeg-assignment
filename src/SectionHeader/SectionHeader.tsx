import SectionHeaderCSS from './SectionHeader.module.css';

export const SectionHeader: React.FC<{label: string}> = ({label}) => {
    return (
      <div className={SectionHeaderCSS.sectionHeader}>{label}</div>
    )
  }