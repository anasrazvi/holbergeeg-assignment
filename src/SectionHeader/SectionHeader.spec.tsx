import renderer from 'react-test-renderer';
import {SectionHeader} from './SectionHeader';

describe("Section Header Component", () => {
    test('renders section header', () => {
      const sectionHeader = renderer
                          .create(<SectionHeader label="Study List:"></SectionHeader>).toJSON();
      expect(sectionHeader).toMatchSnapshot();
    });
  });