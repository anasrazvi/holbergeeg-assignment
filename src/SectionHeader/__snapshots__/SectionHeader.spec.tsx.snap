// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section Header Component renders section header 1`] = `
<div
  className="sectionHeader"
>
  Study List:
</div>
`;
